extensions/blocks/logo-gallery/editor.scss

```diff
@@ -0,0 +1,92 @@
+@import './style.scss';
+
+// Override the default list style type _only in the editor_
+// to avoid :not() selector specificity issues.
+// See https://github.com/WordPress/gutenberg/pull/10358
+ul.wp-block-jetpack-logo-gallery li {
+	list-style-type: none;
+}
+
+.logo-gallery-item {
```

Comment:
You might want to protect this from leaking out to other "logo-gallery-items" by wrapping it in `wp-block-jetpack-logo-gallery`